docs/source/custom_sockets.rst

```diff
@@ -0,0 +1,214 @@
+.. _custom-sockets:
+
+=========================
+Custom Sockets in Syncano
+=========================
+
+``Syncano`` provides possibility of creating the custom sockets. It means that there's a possibility
+to define a very specific endpoints in syncano application and use them as normal api calls.
+Currently custom sockets allow only one dependency - script. This mean that on the backend side
+each time the api is called - the script is executed and result from this script is returned as a result of the
+api call.
+
+Creating a custom socket
+------------------------
+
+There are two methods of creating the custom socket. First: use the helpers objects defined in Python Libray.
+Second: use the raw format - this is described below.
+
+To create a custom socket follow the steps::
```

Comment:
follow ~~the~~ *these* steps
Well, these aren't exactly steps. But more like an example. But if we add some comments below it will be more "steps-like" ;)